examples/Remarketing/AddMerchantCenterDynamicRemarketingCampaign.php

```diff
@@ -274,62 +275,51 @@ private static function createAd(
             $googleAdsClient,
             $customerId,
             'https://goo.gl/mtt54n',
-            'Logo Image'
+            'Square Marketing Image'
         );
 
         // Creates the responsive display ad info object.
         $responsiveDisplayAdInfo = new ResponsiveDisplayAdInfo([
-            'marketing_images' => [
-                new AdImageAsset([
-                    'asset' => new StringValue(['value' => $marketingImageResourceName])
-                ])
-            ],
-            'square_marketing_images' => [
-                new AdImageAsset([
-                    'asset' => new StringValue(['value' => $logoImageResourceName])
-                ])
-            ],
-            'headlines' => [
-                new AdTextAsset([
-                    'text' => new StringValue(['value' => 'Travel'])
-                ])
-            ],
+            'marketing_images' => [new AdImageAsset([
+                'asset' => new StringValue(['value' => $marketingImageResourceName])
+            ])],
+            'square_marketing_images' => [new AdImageAsset([
+                'asset' => new StringValue(['value' => $squareMarketingImageResourceName])
+            ])],
+            'headlines' => [new AdTextAsset([
+                'text' => new StringValue(['value' => 'Travel'])
+            ])],
             'long_headline' => new AdTextAsset([
                 'text' => new StringValue(['value' => 'Travel the World'])
             ]),
-            'descriptions' => [
-                new AdTextAsset([
-                    'text' => new StringValue(['value' => 'Take to the air!'])
-                ])
-            ],
+            'descriptions' => [new AdTextAsset([
+                'text' => new StringValue(['value' => 'Take to the air!'])
+            ])],
             'business_name' => new StringValue(['value' => 'Interplanetary Cruises']),
             // Optional: Call to action text.
             // Valid texts: https://support.google.com/adwords/answer/7005917
             'call_to_action_text' => new StringValue(['value' => 'Apply Now']),
+            // Optional: Sets the ad colors.
+            'main_color' => new StringValue(['value' => '#0000ff']),
+            'accent_color' => new StringValue(['value' => '#ffff00']),
+            // Optional: Sets to false to strictly render the ad using the colors.
+            'allow_flexible_color' => new BoolValue(['value' => false]),
+            // Optional: Sets the format setting that the ad will be served in.
+            'format_setting' => DisplayAdFormatSetting::NON_NATIVE
             // Optional: Creates a logo image and set it to the ad.
-            'logo_images' => [
-                new AdImageAsset([
-                    'asset' => new StringValue(['value' => $logoImageResourceName])
-                ])
-            ],
+            // 'logo_images' => [new AdImageAsset([
+            //     'asset' => new StringValue(['value' => INSERT_LOGO_IMAGE_RESOURCE_NAME_HERE])
```

Comment:
Could you use quote here too: 

```
'INSERT_LOGO_IMAGE_RESOURCE_NAME_HERE'
```

So that when a user uncomment these lines, at least it doesn't result in syntax errors in IDE.